jsdojo/API_Task/javascriptAPI.js

```diff
@@ -0,0 +1,50 @@
+(function() {
+    'use strict';
+    kintone.events.on('app.record.create.show', (event) => {
+        const record = event.record;
+        let i;
+        let dd;
+        let newRow;
+        console.log(event.record.table);
+        record.table.value[0].value.action_5.value = 'Relentless quest';
+        for (i = 0; i < 5; i++) {
```

Comment:
- Instead of using `switch`, you can create an array called `action5`, put all action5 items there, and create rows for each action.
```js
const action5 = [
  'Relentless quest',
  'Mental and physical persistence',
  'Empathy for the ideal',
  'Inspire others',
  'Increase knowledge and skills',
  'Fairness and honesty'
];
const action5Length = action5.length;

for (let i = 0; i < action5Length; i++) {
  //  Create a newRow and push it to the table
}
```